modules/search-service-impl/src/main/java/org/opencastproject/feed/impl/FeedServiceImpl.java

```diff
@@ -94,6 +97,36 @@
   /** The security service */
   private SecurityService securityService = null;
 
+  /** For Feedlinks */
+  private Gson gson = new Gson();
+
+  /*
+   *
+   * Feedlinks for Admin UI
+   * /feeds/feeds
+   *
+   */
+
+  @GET
+  @Path("/feeds")
+  @Produces(MediaType.APPLICATION_JSON)
+  public String listFeedServices() {
+
+    List<Map<String, String>> feedServices = new ArrayList<>();
+
+    for (FeedGenerator generator : feeds) {
+      Map<String, String> details = new HashMap<>();
+      details.put("identifier", generator.getIdentifier());
```

Comment:
The identifier you are getting here is not correct.  I'm seeing this render as (for example) `http://localhost:8080/series`, when it should be rendering as:
* `http://localhost:8080/feeds/rss/2/series/338f6880-2ccc-4869-8133-c53d3c4de430` (for rss)
* `http://localhost:8080/feeds/atom/1/series/338f6880-2ccc-4869-8133-c53d3c4de430` (for atom).

Ideally this method would take a parameter setting the type of feed to return, but if that's not possible then it should return both, or rss.

I get that given the endpoint's parameters you can't know the some of the parameters for the feed (like the series ID), but returning an identifier with a marker for substitution would make sense.  In other words, `http://localhost:8080/feeds/atom/1/series/$seriesid$`, or some other easy-to-parse marker so that the admin ui can replace the marker with the correct series id.

Reply:
so instead of showing a list of wrong data I just need to show the atom/rss links to the series itself? (see picture posted down below)

rss: 
https://develop.opencast.org/feeds/rss/2.0/series/20f4cc55-db54-421d-b593-254b44262cae

atom:
https://develop.opencast.org/feeds/atom/1.0/series/20f4cc55-db54-421d-b593-254b44262cae


instead of this.
![61053502-f6850480-a3ed-11e9-903e-195718176147](https://user-images.githubusercontent.com/20824075/61468185-29467400-a97d-11e9-8814-1d409748792c.png)

So my solution was fundamentally wrong?!

Reply:
No, you are fundamentally right :)  Your initial code is basically right, and actually nice and clean despite my nit picking.  You just need to modify the identifier the service is sending you into something which works when clicked on.  I would leave the feed service alone, and just fix the URLs using some string magic in the UI to render the correct values (like the ones you posted above) in the identifier column.  You might need to do some CSS tricks to wrap the column nicely so it doesn't run off the side of the screen, but you're very very close to this being done!

Reply:
@gregorydlogan I am working on it, I just have not commited by now, I send you an update until I need feedback

Reply:
@gregorydlogan my current feed tab view looks like this, are there links missing? I still need to git commit if this view is right.
![Screenshot from 2019-09-05 09-37-21](https://user-images.githubusercontent.com/20824075/64321504-f30e9380-cfc0-11e9-90a0-fbcbcce1cade.png)

Reply:
Yep, those urls look right to me!

Reply:
@gregorydlogan pls look at #1050, this is a updated one from this